### ISSUE TITLE
Export with alignments fails in tCore which also breaks upload

### ISSUE THREAD

**Issue by PhotoNomad0** [unfoldingWord/translationCore#7511]
### User Story

Export with alignments fails in tCore which also breaks upload. It also affects uploading to D43 (which also exports to USFM before upload).  Seems to be related to verse spans.  Perhaps those that were partially aligned before verse span alignment was added.

To reproduce try exporting this project:

[https://app.zenhub.com/files/65028237/981a55a4-86ab-4300-b4ee-df431be93290/download](https://app.zenhub.com/files/65028237/981a55a4-86ab-4300-b4ee-df431be93290/download)

### Features / Specifications

- [ ]
- [ ]
- [ ]

### Definition of Done

- [ ]
- [ ]
- [ ]

**Comment by PhotoNomad0** [comment]
@elsylambert @danielklapp - The fix is in translationCore 3.5.0 (30831db).

When testing with the above project - if before opening the project, you export to USFM with alignments.  Then you will now see a warning that alignments have been invalidated.  If you click on export.  Then will see that alignments being updated for Num 5:12, 5:13, and 5:20.  And then will get the file dialog to select where to save the file.

**Comment by PhotoNomad0** [comment]
**Note:** 

the exceptions thrown in old tC versions are related to the empty verses in ch 5 - 12,13,20,21

**Comment by elsylambert** [comment]
@PhotoNomad0 Testing in translationCore 3.5.0 (30831db) with the above mentioned steps and  the above project. When exported to USFM with alignments., before opening the project, saw a warning that alignments have been invalidated>clicked on export and then saw that alignments were updated for Num 5:12, 5:13, and 5:20. After that I got the file dialog to select where to save the file.
<img width="1438" alt="Screenshot 2023-07-11 at 10 38 44 AM" src="https://github.com/unfoldingWord/translationCore/assets/51806180/5519a60b-f5bb-407a-a9bb-8bfd43a17349">
